src/ansys/fluent/core/session.py

```diff
@@ -133,10 +133,20 @@ def _build_from_fluent_connection(
         if self._start_transcript:
             self.transcript.start()
 
+        if FluentVersion(self.scheme_eval.version) < FluentVersion.v252:
+            self._app_utilities = AppUtilitiesOld(self.scheme_eval)
+        else:
+            self._app_utilities = (
+                self._fluent_connection._connection_interface._app_utilities
+            )
```

Comment:
```suggestion
        # TODO app_utilities module to be the APPUtilities factory
        # so that these complications will be hidden from calling code.
        if FluentVersion(self.scheme_eval.version) < FluentVersion.v252:
            self._app_utilities = AppUtilitiesOld(self.scheme_eval)
        else:
            self._app_utilities = (
                self._fluent_connection._connection_interface._app_utilities
            )
```

Reply:
@seanpearsonuk We have created an issue to track this.